src/preload.js

```diff
@@ -407,7 +407,21 @@ setInterval(function () {
     if (store.get(skipArtists)) {
       const skippedArtists = store.get(settings.skippedArtists);
       if (skippedArtists.find((artist) => artist === artists) !== undefined) {
-        elements.click("next");
+        if (artists === "TIDAL") {
```

Comment:
Please put this behind a feature flag in the settings & extract it to a function.
Other than that it looks good. 😄 


*Additional*

One boy scout fix that you could also make (but not required) in the rest of the code is bundling the 2 mute lines into a function:

```typescript
setMuted(muted: boolean){
   elements.click("volume");
   isMutedArtist = muted;
}
```
You can then use it in this part of the code but also in the rest of the code:

```typescript
          if (!elements.isMuted()) {
              setMuted(true);
          }
```

Reply:
Fixed it. See commit message for more details.